### ISSUE TITLE
Complete NZBGet Switch Custom Component

### ISSUE THREAD

**Issue by stanvx** [stanvx/Home-Assistant-Configuration#2]
Need to Complete/Fix NZBGet Switch Custom Component to allow the Pause and Resume functions for downloads.

https://github.com/stanvx/Home-Assistant-Configuration/blob/master/custom_components/switch/nzbget.py

**Comment by stanvx** [comment]
Switch will show the correct status of On/Off although it doesn’t update and the toggle doesn’t work as it should. The core code including the JSON-RPC wrapper has been taken from the NZBGet Sensor Component.

API Reference: https://nzbget.net/api/

**Comment by stanvx** [comment]
Fixed: commit 5185ccd7d830a98497f162c5857622aaa994b08d

Utilized command_line switch instead of custom component:

```
command_on: "curl -XPOST http://username:password@0.0.0.0:0000/jsonrpc -d '{\"method\": \"pausepost\"}' && curl -XPOST http://username:password@0.0.0.0:0000/jsonrpc -d '{\"method\": \"pausedownload\"}'"
command_off: "curl -XPOST http://username:password@0.0.0.0:0000/jsonrpc -d '{\"method\": \"scheduleresume\", \"params\": [1]}'"
command_state: "curl -s http://username:password@0.0.0.0:0000/jsonrpc -d '{\"method\": \"status\"}' |grep 'ServerPaused\" : true'" 
```
